packages/odyssey-react-mui/src/labs/UiShell/UiShellContent.tsx

```diff
@@ -0,0 +1,146 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import styled from "@emotion/styled";
+import { memo, type ReactNode } from "react";
+import { ErrorBoundary, ErrorBoundaryProps } from "react-error-boundary";
+
+import { SideNav, type SideNavProps } from "../SideNav";
+import { TopNav, type TopNavProps } from "../TopNav";
+import {
+  useOdysseyDesignTokens,
+  type DesignTokens,
+} from "../../OdysseyDesignTokensContext";
+
+const AppContainer = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>(({ odysseyDesignTokens }) => ({
+  overflowX: "hidden",
+  overflowY: "scroll",
+  paddingBlockEnd: odysseyDesignTokens.Spacing4,
```

Comment:
We can remove all these logical selectors. 
```suggestion
  padding: odysseyDesignTokens.Spacing4,
```

Reply:
Why's that? We're adding padding inside UI Shell to match Top Nav. Apps can have their own padding, but this is going to match us to the exact padding required for the new UI Shell.